Jenkinsfile

```diff
@@ -75,7 +75,7 @@ pipeline {
     stage ('Build') {
       steps {
         wrap([$class: 'Xvnc', takeScreenshot: false, useXauthority: true]) {
-          withMaven(jdk: 'adoptopenjdk-hotspot-jdk8-latest', maven: 'apache-maven-latest', options: [junitPublisher(disabled: true), openTasksPublisher(disabled: true)]) {
+          withMaven(jdk: 'jdk11', maven: 'apache-maven-latest', options: [junitPublisher(disabled: true), openTasksPublisher(disabled: true)]) {
```

Comment:
we still want to have java 8 by default.
we should do somthing similar with hybrid builds/target platforms as we do at xtext